exporter/azuremonitorexporter/trace_to_envelope.go

```diff
@@ -106,6 +105,12 @@ func spanToEnvelope(
 		data.BaseType = remoteDependencyData.BaseType()
 	}
 
+	// Record the Span status message as a property if there
+	statusMessage := span.Status().Message()
+	if len(statusMessage) > 0 {
+		dataProperties["Status.message"] = statusMessage
```

Comment:
Can we define `Status.message` as const? This property key would be good to start with lowercase like the others.

Also can we add simple unit-test for this property?

Reply:
After chatting we agreed to pass the raw Span status fields through as properties:

Status.Code => otel.status_code
Status.DeprecatedCode => otel.status_deprecatedcode
Status.Message => otel.status_description (description is what is spec'd)